assets/src/web-stories-block/block/block-types/latest-stories/edit.js

```diff
@@ -53,12 +52,10 @@ const LATEST_STORIES_QUERY = {
  * @param {Function} root0.setAttributes Callable function for saving attribute values.
  * @return {*} JSX markup for the editor.
  */
-const LatestStoriesEdit = ({ attributes, setAttributes }) => {
+function LatestStoriesEdit({ attributes, setAttributes }) {
```

Comment:
Why change this?

Reply:
Answered at https://github.com/google/web-stories-wp/pull/8339#discussion_r671268346